sql/sql_expression_operations.cc

```diff
@@ -2256,21 +2256,48 @@ bindBuiltinFunction(SqlBindingScope & scope,
                 this,
                 fn.resultInfo};
     }
-    else {
+
+    if (functionName == "try") {
         return {[=] (const SqlRowScope & row,
                      ExpressionValue & storage,
                      const VariableFilter & filter) -> const ExpressionValue &
-                {
-                    std::vector<ExpressionValue> evaluatedArgs;
-                    evaluatedArgs.reserve(boundArgs.size());
-                    for (auto & a: boundArgs)
-                        evaluatedArgs.emplace_back(std::move(a(row, fn.filter)));
+            {
+                ExcAssert(args.size() == 2);
+                std::vector<ExpressionValue> evaluatedArgs;
+                evaluatedArgs.reserve(boundArgs.size());
+                try {
+                    evaluatedArgs.emplace_back(std::move(
+                        boundArgs[0](row, fn.filter)));
+                }
+                catch (const ML::Exception & exc) {
```

Comment:
The HttpReturnException has a lot more information within it, which we should extract here.  Or at least use ML::getExceptionString().